src/app/space/settings/areas/create-area-dialog/create-area-dialog.component.ts

```diff
@@ -26,8 +32,10 @@ export class CreateAreaDialogComponent implements OnInit {
   @ViewChild('rawInputField') rawInputField: ElementRef;
   @ViewChild('inputModel') inputModel: NgModel;
 
+  AreaCreationStatus: typeof AreaCreationStatus = AreaCreationStatus;
```

Comment:
This looks kind of funky. What's it for?

Reply:
This line is declaring the enum so it will be defined for use in the HTML

Reply:
Ah, it's so the template can reference enum members by name. I see. Could you add a comment explaining that? Without that context, just seeing a component property named and declared this way is pretty strange. Alternatively, a string enum could avoid this kind of thing:

```
enum AreaCreationStatus {
  OK = 'OK',
  ERROR = 'ERROR'
}
```

```
<span *ngIf="status == 'ERROR'">Got an error</span>
```

Reply:
Yeah that's fair, it does look pretty odd without context - I've added a comment for it.